templates/addons/cilium/cilium.yaml

```diff
@@ -23,6 +23,11 @@ spec:
     tunnelProtocol: ""
     enableIPv4Masquerade: true
     egressMasqueradeInterfaces: eth0
+    policyAuditMode: ${FW_AUDIT_ONLY:=true}
```

Comment:
Should we set default to false to enable firewall by default as we want to protect clusters by default.